index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
+    <title>Document</title>
+</head>
+<body>
+    <header>
+        <div class="head-content">
+            <p class="logo">
+               <span>Welcome</span>
+            </p>
+            <p class="nav">
+                <span></span>
+                <span></span>
+                <span></span>
+            </p>
+        </div>
+    </header>
+    <div class="presentation">
```

Comment:
- [ ] Nice working on your Headline, kindly use a `section` tag instead of a `div` for better use of HTML semantics.

Reply:
thank you, dear!